DependencyInjection/HttplugExtension.php

```diff
@@ -56,6 +61,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_journal_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.plugin.history');
```

Comment:
ah, got it. maybe we should use a more specific service name for this. its not "the" history plugin, but a history plugin linked to the message journal.